src/View/Helper/TimeHelper.php

```diff
@@ -326,8 +326,12 @@ public function format($date, $format = null, $invalid = false, $timezone = null
      * @throws \InvalidArgumentException When the date cannot be parsed
      * @see \Cake\I18n\Time::i18nFormat()
      */
-    public function i18nFormat($date, $format = null, $invalid = false, $timezone = null)
+    public function i18nFormat($date = null, $format = null, $invalid = false, $timezone = null)
```

Comment:
Why are you changing the signature? This first argument should not be optional for your changes to make sense.
`= null` would only make sense with the opposite, of null = now.